SYCL/AtomicRef/atomic_memory_order_acq_rel.cpp

```diff
@@ -1,33 +1,239 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -O3 -o %t.out -Xsycl-target-backend=nvptx64-nvidia-cuda --cuda-gpu-arch=sm_70
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 // L0, OpenCL, and HIP backends don't currently support
 // info::device::atomic_memory_order_capabilities
-// UNSUPPORTED: level_zero || opencl || hip
+// UNSUPPORTED: level_zero, opencl, hip
 
-// NOTE: Tests load and store for acquire-release memory ordering.
+// host does not support barrier
+// XFAIL: host
```

Comment:
Maybe we just remove the %HOST_RUN_... line?

Reply:
It only shaves of 0.2 seconds.